components/Dialogs/src/private/MDCAlertControllerView+Private.m

```diff
@@ -353,8 +355,22 @@ - (CGSize)calculateActionsSizeThatFitsWidth:(CGFloat)boundingWidth {
 - (void)layoutSubviews {
   [super layoutSubviews];
 
-  for (UIButton *button in self.actionButtons) {
+  for (MDCFlatButton *button in self.actionButtons) {
```

Comment:
Since MDCFlatButton is scheduled for deprecation, do you want to make this an `MDCButton *` cast instead?